app/login/page.tsx

```diff
@@ -67,27 +118,88 @@ export default function Login() {
             )}
           </div>
 
-          <div>
-            <Label htmlFor="password" className="text-xs uppercase">
-              Password
+          {loginMethod === 'password' ? (
```

Comment:
would it be possible to separate the login with password and login with otp components. the stacked (?:) operators gets a bit confusing to follow